test/parallel/test-config-json-schema.js

```diff
@@ -0,0 +1,72 @@
+// Flags: --no-warnings --expose-internals
+
+'use strict';
+
+require('../common');
+
+const {
+  getEnvOptionsInputType
+} = require('internal/options');
+const currentJsonSchema = require('../../doc/node_config_json_schema.json');
+const assert = require('assert');
+
+const map = getEnvOptionsInputType();
+
+function parseValueFlag(value) {
```

Comment:
Can you add as much as comment possible to this file? I'm having hard time reviewing it.